onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -388,4 +394,34 @@ public void checkEditOnlineWithEditPermission() {
     addUsers.deleteUser(userC_name);
     manageLogInOut.signOut();
   }
+
+
+
+  @Test
+  /**
+   * Save_version_drawer_US01
+   * Check that "Add new version" button existing and that the drawer is opened
+   */
+  public void checkOpeningDrawerVersions() {
+    String document = "OO_test";
+    String extension = ".docx";
+    String userName = PLFData.DATA_NAME_ROOT;
+    info("Edit Online from document from AS");
+    activityStream.uploadFileFromAS(document + extension);
+    homePagePlatform.refreshUntil($(byXpath(ELEMENT_CHECK_FILE_IS_UPLOADED.replace("$doc", document + extension))), visible, timeout);
```

Comment:
Do we really need to refresh the page here ? When I post a file manually I don't need to refresh, I just need to wait a bit.